srcpkgs/fonts-droid-ttf/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'fonts-droid-ttf'
+pkgname=fonts-droid-ttf
+version=20150613
+revision=1
+noarch=yes
+_githash=f5de525ee3547b8a69a21aec1e1a3175bc06f442
+create_wrksrc=yes
+depends="font-util"
+font_dirs="/usr/share/fonts/TTF"
+short_desc="General-purpose Android fonts (CJK support)"
+maintainer="travankor <travankor@tuta.io>"
+license="Apache-2.0"
+homepage="http://www.droidfonts.com/droidfonts/"
+changelog="https://android.googlesource.com/platform/frameworks/base/+log/master/data/fonts"
+distfiles="https://android.googlesource.com/platform/frameworks/base/+archive/${_githash}/data/fonts.tar.gz"
+checksum=@722c954d839cc131c17161956c5117624ceeea3f90e01d4d8ad3d38f374000cc
```

Comment:
`checksum=@` ??

Reply:
checksum=@ is used when the code is generated on the fly when downloaded so it changes everytime we fetch it, prefixing @ forces it to check the hash of the content.

Reply:
@maxice8 I didn't know that, thank you!